core/posts.go

```diff
@@ -493,14 +523,37 @@ func validatePost(post *pb.Post) (err error) {
 
 	// Tags
 	if len(post.Tags) > MaxPostTags {
-		return fmt.Errorf("Tags in the post is longer than the max of %d characters", MaxPostTags)
+		return fmt.Errorf("Tags in the post is longer than the max of %d", MaxPostTags)
```

Comment:
Export this error, too. (And all the rest... you get the idea.)

Reply:
Done.